math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -968,7 +2039,7 @@ template <typename T>
 auto Any(const RVec<T> &v) -> decltype(v[0] == true)
 {
    for (auto &&e : v)
-      if (e == true)
+      if (static_cast<bool>(e) == true)
```

Comment:
Why not just `if (e)`?

Reply:
i have no idea, this code is from RVec 1.0. removed it, thanks